doc/getting_going.rst

```diff
@@ -122,6 +122,35 @@ latest version simply do:
 
    > pip install fparser --upgrade
 
+git submodule
++++++++++++++
+
+Although PSyclone releases always work with a released version of
```

Comment:
I think I would start explaining that users might want to test or use the latest features of PSyclone and that these are available on master. Then say that whilst releases always work with other released software and can therefore be installed simply with pip, the same is not necessarily true for master as some development is currently done concurrently with fparser. Therefore to support users (and developers) the submodule feature is used to ensure the appropriate version of fparser is used for a particular version of PSyclone. ...